src/framework/acl/core.clj

```diff
@@ -17,6 +17,12 @@
   ([uri]
    (re-find #"\w+" uri)))
 
+(defn- user->role
+  [u]
+  (when
+    (or (:users/is_active u) (:is_active u))
```

Comment:
Just curiosity, why underscoring?

Reply:
It's the raw db response, without any transformation. Maybe later we should create a db->model transformer.